source/llvm/LLVMModelDataSymbols.h

```diff
@@ -357,7 +357,7 @@ class LLVMModelDataSymbols
     bool isNamedSpeciesReference(const std::string& id) const;
 
     const SpeciesReferenceInfo& getNamedSpeciesReferenceInfo(
-            const std::string& id) const;
+            const std::string& id);
```

Comment:
If we go with the suggestion n the previous comment I made, we don't need to remove the const of this function.